CODE_OF_CONDUCT.md

```diff
@@ -18,47 +48,63 @@
+
+* Do not distribute sexual or violent imagery without a Content Warning.
+* Do not distribute any sexualized depictions of minors, in any way. (This includes drawings and 3D renders.)
+
+## Where does this CoC apply and who does this CoC apply to?
 
-## Where does this CoC apply and who does this CoC apply to? 
 This code of conduct applies to all official public discussion forums related to this project, including but not limited to public chat, forums, and comments on code.
 
 This code applies to any member of the community participating in discussions, contributing to the project, or otherwise discussing the project publicly.
 
 It also would apply in the future main Florence instance. Members of the instance must agree to the CoC.
-## What if someone violates this Code of Conduct while outside of Florence instances, or in a space or medium to which the Code does not apply? 
+
+## What if someone violates this Code of Conduct while outside of Florence instances, or in a space or medium to which the Code does not apply?
 
 Use your judgement, but always feel free to let us know -- we’d rather know more than less. If the person who violates the Code of Conduct intersects with the Florence community in any way, we encourage you to make a report, even if the offending behavior itself was outside of our space.
-## When Something Happens 
+
+## When Something Happens
+
 If you see behavior that is not aligned with this Code of Conduct, here’s how you can handle it.
 
 Consider the situation and if it might be useful to screenshot messages in case the messages in question are deleted.
 
-The person may not be completely aware of the Code of Conduct or specific items within it. *If you consider it safe* and you are comfortable in doing so you can start by letting them know that their actions weren’t appropriate and point them to specific parts of the Code of Conduct. One suggestion is to send a private message first, because this can avoid potentially embarrassing someone. Politely and patiently let the person know that their behavior isn’t in line with the Code of Conduct. Share the code of conduct with them and refer to specific parts which you think their behavior is not in line with. You may ask them to adjust their behavior or possibly edit or delete a message. 
+The person may not be completely aware of the Code of Conduct or specific items within it. **If you consider it safe** and you are comfortable in doing so you can start by letting them know that their actions weren’t appropriate and point them to specific parts of the Code of Conduct. One suggestion is to send a private message first, because this can avoid potentially embarrassing someone. Politely and patiently let the person know that their behavior isn’t in line with the Code of Conduct. Share the code of conduct with them and refer to specific parts which you think their behavior is not in line with. You may ask them to adjust their behavior or possibly edit or delete a message.
```

Comment:
May want to re-do apostrophes through this doc. ’ looks similar to ' but they may not be interpreted correctly by screen readers or translation programs.

Reply:
Checked updated version after pull request and this still seems to be an issue. If you are using a text editor you should be able to do a find and replace.

Reply:
Scratch that. Was just Github being silly, even after I refreshed the page! Resolving